src/extra/gfx/feature.cpp

```diff
@@ -81,22 +91,27 @@ struct BuiltinFeatureShard {
 };
 
 struct FeatureShard {
-  /* Input table
-    {
-      :Shaders [
-        {:Name <string> :Stage <enum> :Before [...] :After [...] :EntryPoint (-> ...)}
-      ]
-      :State {
-        :Blend {...}
-      }
-      :DrawData [(-> ...)]/(-> ...)
-      :Params [
-        {:Name <string> :Type <type> :Dimension <number>}
-      ]
-    }
-  */
   static SHTypesInfo inputTypes() { return CoreInfo::AnyTableType; }
   static SHTypesInfo outputTypes() { return Types::Feature; }
+  static SHOptionalString help() {
```

Comment:
There is a serious discussion here to be made about embedded docs using markdown or not. I think it's time to have it.
I'd say yes as it would help current generation. 
@Kryptos-FR also our current UI shards should easily support markdown right?

Reply:
Yeah was thinking we might want to show some more elaborate help strings, I know this package does at least some basic markdown like code blocks: https://github.com/lampsitter/egui_commonmark

Reply:
let's implement https://github.com/lampsitter/egui_commonmark